internal/compiler/tests/syntax/imports/import_errors.slint

```diff
@@ -10,7 +10,7 @@ import { NotExported } from "../../typeloader/incpath/local_helper_type.slint";
 import { Nothing } from "";
 //                      ^error{Unexpected empty import url}
 
-import "invalid_export.slint";
+import "../../typeloader/incpath/local_helper_type.slint";
```

Comment:
This seemingly unrelated change was done to avoid an early error-out due to the file not existing. We're not interested in tracking the error of a non-existent file, we're interested in tracking the error when not import names are specified.